### ISSUE TITLE
Is this abandoned?

### ISSUE THREAD

**Issue by zomars** [ReactifyJS/Reactify#1]
I would like to try and upgrade the dependencies and maintain this stack if that's ok.

**Comment by SachaG** [comment]
Yeah this was more of a fun experiment/proof of concept to hack on. I don't use Shopify myself at the moment so I can't really work on it more… Would you like me to transfer you the repo ownership?

**Comment by zomars** [comment]
That would be nice of you. Thank you!

El 12 jun 2019, a la(s) 0:06, Sacha Greif <notifications@github.com> escribió:

Yeah this was more of a fun experiment/proof of concept to hack on. I don't use Shopify myself at the moment so I can't really work on it more… Would you like me to transfer you the repo ownership?

**Comment by impactcolor** [comment]
@zomars did you take over this?

**Comment by zomars** [comment]
@impactcolor  not that I’m aware of

**Comment by SachaG** [comment]
Sorry, I completely forgot! It should be yours now :)

**Comment by SachaG** [comment]
Hmm I wasn't able to transfer the thing to you completely but I did make you owner of the org. Hopefully that's enough.
